lib/msal-common/src/response/ResponseHandler.ts

```diff
@@ -192,7 +192,7 @@ export class ResponseHandler {
             if (
                 refreshAccessToken &&
                 serverResponse.status &&
-                serverResponse.status >= HttpStatus.SERVER_ERROR_RANGE_START &&
+                serverResponse.status >= HttpStatus.INTERNAL_SERVER_ERROR &&
```

Comment:
Inconsistent with the "SERVER_ERROR_RANGE_END". You can have 2 constants for 500. Or just hardcode these error codes, they are well known

Reply:
I don't think we should hardcode. No "magic numbers", right? (even though it's obvious)

Reply:
I do like the idea of having two different constants for 500